pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name = "perceval"
+version = "0.17.1"
+description = "Fetch data from software repositories"
+authors = [
+    "GrimoireLab Developers"
+]
+license = "GPL-3.0+"
+
+readme = "README.md"
+
+homepage = "https://chaoss.github.io/grimoirelab/"
+repository = "https://github.com/chaoss/grimoirelab-perceval"
+
+keywords = [
+    "development",
+    "grimoirelab"
+]
+
+packages = [
+    { include = "perceval" },
+    { include = "tests", format = "sdist" },
+]
+
+classifiers = [
+   "Development Status :: 4 - Beta",
+   "Intended Audience :: Developers",
+   "Intended Audience :: Science/Research",
+   "Topic :: Software Development",
+   "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+   "Programming Language :: Python :: 3"
+]
+
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/chaoss/grimoirelab-perceval/issues"
+
+[tool.poetry.scripts]
+perceval = 'bin.perceval:main'
+
+[tool.poetry.dependencies]
+python = "^3.6"
```

Comment:
Do you want me to mention the various versions here as we did in ELK [pyproject.toml#L41](https://github.com/chaoss/grimoirelab-elk/blob/master/pyproject.toml#L41)?
I feel this is enough, but what are your thoughts on this?

Reply:
I think that's enough. It will work for `3.x`.